### DISCUSSION TITLE
Videos not playing

### DISCUSSION THREAD

**Discussion by Roucseba** [photoprism/photoprism#3546] [Q&A] [answered]
Hi,
I am new to Photoprism. It works well on photos, but I don't see any videos Iuploaded. 
I see them in the folders when I connect with WebDAV, but they just don't appear in my Photoprism library on the search tab, video tab etc.
Seeing the logs, I read an error line such as` 2023-07-13 12:05:46 ERRO import: ffmpeg version 5.1.2-3ubuntu1 Copyright (c) 2000-2022 (...)` 

I am on PhotoPrism® Plus Build 230625-17242fb07, hosted on Pikapods 2 CPUcores, 8 GB memory, 50 GBstorage.
The FFmpeg anf Exiftools are not desactivated on the Advanced menu.

Thank you in advance!

**Comment by lastzero** [comment]
Although it is theoretically possible, I wouldn't expect this to be caused by an upload size limit.

- To be sure, you could repeat the test with smaller files to see if it works then.
- If you have some non-private files available, you could also test on our public demo at https://demo.photoprism.app/ to see if it works.

If none of this works and you think it might be a file format issue, feel free to email us sample files for debugging:

https://www.photoprism.app/contact#file-samples

**Reply by Roucseba** [reply]
Great, thank you!
I did a 10 sec video, and it works. It appears in the library and I can play it.
What are the criteria for upload size limit? How can I unlock it?
Thanks

**Reply by lastzero** [reply]
You'd have to ask the PikaPods support for details on the upload limit, if one exists.

**Reply by Roucseba** [reply]
OK, will do. Thank you!

**Reply by Roucseba** [reply]
Hi again,

I have their reply:

### 

> Hi,
> 
> Thank you for choosing Pikapods for your Photoprism hosting. Regarding the upload size limit, Pikapods does not impose any specific limits on uploads. However, there might be limitations within Photoprism itself or other factors that could affect uploading larger videos.
> 
> In such cases, a possible workaround is to upload the videos via SFTP into the import folder of your Photoprism app. This bypasses any potential size restrictions imposed by the web interface.
> 
> If you're experiencing difficulties with video uploads, it's recommended to check your network stability and ensure that the videos are in a supported format and codec by Photoprism.
> 
> Additionally, it's advisable to continue working with the Photoprism support team on GitHub, as they have in-depth knowledge of the application and can provide specific guidance to resolve any issues you may encounter.
> 
> If you have any further questions or need further assistance, please let us know.
> 
> 
> Thanks,
> The PikaPods Team 

And maybe it makes sense, since I can transfer my files by webDAV and they are in the folder.
I found the menu Libray -> errors and then various entries :
index: 2023/07/20230711_030750_39E3A093.mp4 exceeds file size limit (2.2 GB / 1.0 GB) 

So it seems there is a 1 GB limit.
I found in the documentation the PHOTOPRISM_ORIGINALS_LIMIT critera.
I change it to -1 instead of 1000.
I will see if it works.
Thanks again

**Reply by lastzero** [reply, marked as answer]
In that case, simply increase the default file size limit (in MB) by setting a higher value with the PHOTOPRISM_ORIGINALS_LIMIT variable:

https://docs.photoprism.app/getting-started/config-options/#storage

**Comment by Roucseba** [comment]
Hi,
I confirm it works!
I told PikaPods about it and they updated their FAQ about this https://docs.pikapods.com/apps/photoprism/#maximum-upload-size
Thanks again!

**Reply by lastzero** [reply]
Glad you got it working! :)
